### PR TITLE
APPS-11107 - escape video tag id before passing to querySelector

### PR DESCRIPTION
**Pull request by aleksandar-cloudinary**
https://cloudinary.slack.com/archives/C9B1DMFV3/p1638468005034600

### CODE REVIEW

File: src/video-player.utils.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import {
   AUTO_PLAY_MODE
 } from './video-player.const';
 import { isString } from './utils/type-inference';
+require('css.escape');
```
Comment (e1adn) [inline src/video-player.utils.js:12]: Please change it to:
`import cssEscape from 'css.escape';`

File: src/video-player.utils.js [hunk 2]
```diff
@@ -33,7 +34,7 @@ export const getResolveVideoElement = (elem) => {
     }
 
     try {
-      elem = document.querySelector(`#${id}`) || videojs.getPlayer(id);
+      elem = document.querySelector(`#${CSS.escape(id)}`) || videojs.getPlayer(id);
```
Comment (e1adn) [inline src/video-player.utils.js:37]: and use the imported function : 
`cssEscape (id)`

File: src/video-player.utils.js (hunk 1 above)
Comment (e1adn) [reply src/video-player.utils.js:12]: Also please add a comment about it, something like:  'includes a polyfill to support ie11'

Comment (aleksandar-cloudinary) [reply src/video-player.utils.js:12]: Done

**Review by e1adn — APPROVED** [review]

**Pull request merged**